lib/service/Service.ts

```diff
@@ -141,14 +141,22 @@ class Service {
   }
 
   /**
-   * Add an order to the order book.
+   * Add a limit order to the order book.
    */
-  public placeOrder = async (order: OwnOrder) => {
```

Comment:
I think we could keep the 2 calls (limit/market) unified on `gRPC`/`Service`, and let them split only when `OrderBook` API is being called. same as you did on CLI, based on if price is specified or not.

@sangaman @kilrau